src/test/acceptance/features/do-you-have-children.feature

```diff
@@ -3,28 +3,37 @@ Feature: Do you have children who are 3 years old or younger?
   As a potential claimant
   I want to select whether or not I have children who are 3 years old or younger
 
-  Background:
-    Given I have entered my details up to the do you have children three or younger page
-
   Scenario: Yes and No are displayed
+    Given I have entered my details up to the do you have children three or younger page
     Then Yes and No options are displayed on the do you have children three or younger page
 
   Scenario: Do not select an option on the do you have children three or younger page
+    Given I have entered my details up to the do you have children three or younger page
     When I do not select an option
     And I click continue
     Then I am informed that I need to select an option for do you have children three or younger
 
   Scenario: Select the No option on the do you have children three or younger page
+    Given I have entered my details up to the do you have children three or younger page
     When I say No to the do you have children three or younger question
     And I click continue
     Then I am shown the are you pregnant page
 
   Scenario: Select the Yes option and I am asked to enter my children's dates of birth
+    Given I have entered my details up to the do you have children three or younger page
     When I say Yes to the do you have children three or younger question
     And I click continue
     Then I am shown the enter your childrens dates of birth page
 
   Scenario: Children’s dates of birth is not navigable via the back button if I’ve said I have no children
+    Given I have entered my details up to the do you have children three or younger page
     When I have said No to the do you have children three or younger question
     And I am shown the are you pregnant page
     Then The back link points to the Do you have children who are three years old or younger page
+
+  Scenario: I can change my answer to yes for 'Do you have children?' from the check details page
```

Comment:
As discussed, let's move this out into its own feature file, something like `change-children.feature` then we can reinstate the `Background` for this feature file.